tests/basic/pool_function_test.go

```diff
@@ -920,84 +938,84 @@ var _ = Describe("{StorageFullPoolExpansion}", func() {
-		var testrailID = 50643
-		// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/50643
-
-		StartTorpedoTest("DMThinPoolExpandBeyond15TiBLimit",
-			"Initiate pool expansion using add-disk to 20 TiB target size", nil, testrailID)
-		isDMthin, err := IsDMthin()
-		dash.VerifyFatal(err, nil, "error verifying if set up is DMTHIN enabled")
-		dash.VerifyFatal(isDMthin, true, "DMThin/PX-Storev2 is not enabled on underlaying PX cluster. Skipping `PoolExpandTestBeyond15TiBLimit` test.")
-
-		targetSizeTiB := uint64(20)
-		targetSizeInBytes = targetSizeTiB * units.TiB
-		targetSizeGiB = targetSizeInBytes / units.GiB
-		log.InfoD("Trying to expand pool %s to %v TiB with type add-disk",
-			poolIDToResize, targetSizeTiB)
-		err = Inst().V.ExpandPool(poolIDToResize, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, targetSizeGiB, true)
-		dash.VerifyFatal(err != nil, true, "DMThin pool expansion to 20 TB should result in error")
 	})
 })
 
 
+//var _ = Describe("{PoolExpandTestLimits}", func() {
+//	BeforeEach(func() {
+//		contexts = scheduleApps()
+//	})
+//
+//	JustBeforeEach(func() {
+//		poolIDToResize = pickPoolToResize()
+//		log.Infof("Picked pool %s to resize", poolIDToResize)
+//		poolToResize = getStoragePool(poolIDToResize)
+//	})
+//
+//	JustAfterEach(func() {
+//		AfterEachTest(contexts)
+//	})
+//
+//	AfterEach(func() {
+//		appsValidateAndDestroy(contexts)
+//		EndTorpedoTest()
+//	})
+//
+//	It("Initiate pool expansion (DMThin) to its limits (15 TiB)", func() {
+//		var testrailID = 51292
+//		// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/51292
+//
+//		StartTorpedoTest("PoolExpandTestWithin15TiBLimit",
+//			"Initiate pool expansion using resize-disk to 15 TiB target size", nil, testrailID)
+//
+//		// To achieve total pool size of 15 TiB:
+//		// 1. Add another drive of same size for pool to have 2 drives.
+//		// 2. Perform resize-disk operation which is faster than pool rebalance
+//		//    due to adding a new 7 TiB drive.
+//		targetSizeGiB := (poolToResize.TotalSize / units.GiB) * 2
+//
+//		log.InfoD("Next trying to expand the pool %s to %v GiB with type add-disk",
+//			poolIDToResize, targetSizeGiB)
+//		triggerPoolExpansion(poolIDToResize, targetSizeGiB, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK)
+//		resizeErr := waitForOngoingPoolExpansionToComplete(poolIDToResize)
+//		dash.VerifyFatal(resizeErr, nil, "Pool expansion should not result in error")
+//		verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
+//
+//		targetSizeTiB := uint64(15)
+//		targetSizeInBytes = targetSizeTiB * units.TiB
+//		targetSizeGiB = targetSizeInBytes / units.GiB
+//
+//		log.InfoD("Current Size of the pool %s is %d GiB. Trying to expand to %v TiB with type resize-disk",
+//			poolIDToResize, targetSizeGiB, targetSizeTiB)
+//		triggerPoolExpansion(poolIDToResize, targetSizeGiB, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK)
```

Comment:
Why are we removing this test? We should only remove Add disk test.

Reply:
Because a set up operation of this test (line 980) involves add-disk